### ISSUE TITLE
Stable version of the docs says it's the latest

### ISSUE THREAD

**Issue by jghauser** [papis/papis#942]
The banner in the docs that informs users that they're seeing the latest version of the docs (and should follow a link to access the stable version) is shown on both the latest and stable version.

**Comment by alexfikl** [comment]
Unfortunately, there's no (easy?) way to add different text to the stable / latest versions of the docs since they just clone the repo and use whatever's there.

We'd need to configure some addons (the [Non-stable notification](https://docs.readthedocs.io/en/stable/versions.html) one) to add a little banner that's more helpful:
https://docs.readthedocs.io/en/stable/addons.html

**Comment by jghauser** [comment]
Yeah I thought that this is probably why I can see it both versions. I think in this case it might be nice to use an an Addon after all as the current situation is a bit confusing.
